recipes/python-interface/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "python-interface" %}
+{% set version = "1.6.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "b26a1c7fef45d32d473c9047b07618c66c66e2351f1ff6f71136bccf77f6c123" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+    - funcsigs  # [py2k]
+    - typing  # [py2k or py34]
+
+test:
+  imports:
+    - interface
+    - interface.tests
+  requires:
+    - pytest >=2.8.5
+    - pytest-cov >=1.8.1
+    - pytest-pep8 >=1.0.6
+
+about:
+  home: https://github.com/ssanderson/python-interface
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
```

Comment:
Since you are manually packaging the license file, please add the following comment as a reminder to check whether the license file is included in future releases in the source distribution:

```yaml
...
license_family: APACHE
# License file manually packaged. See https://github.com/ssanderson/python-interface/pull/40
license_file: LICENSE
```

I opened a PR at https://github.com/ssanderson/python-interface/pull/40 to try to deal with this